src/shipit_pulse_listener/default.nix

```diff
@@ -24,6 +24,29 @@ EOF
     '';
   });
 
+  mkBot = branch:
+    let
+      secretsKey = "repo:github.com/mozilla-releng/services:branch:" + branch;
+    in
+      mkTaskclusterHook {
+        name = "Triggers Taskcluster hooks on pulse messages";
+        owner = "babadie@mozilla.com";
+        schedule = [ "0 0 * * * *" ];  # every hour
```

Comment:
from IRC

```
12:31:10     garbas │ since it is a service which should always be on                                                                                   │
12:31:14          ¬ │ on heroku                                                                                                                         │
12:31:21          ¬ │ like other services                                                                                                               │
12:31:32          ¬ │ i mean, i actually like this option you did                                                                                       │
12:31:45          ¬ │ since it will bring our heroku bill down                                                                                          │
12:32:11          ¬ │ and opens a possibility to use taskcluster as a deployment tool :)                                                                │
12:32:18    bastien   the thing is it will be thrown away in 3 months.                                                                                  │
12:32:44          ¬ │ as taskcluster will support natively the pulse messages as hook triggers                                                          │
12:32:55     garbas │ aha i see. then ignore me. you probably told me this, but i forgot it  
```